docs/site/Relation-generator.md

```diff
@@ -0,0 +1,110 @@
+---
+lang: en
+title: 'Relation generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Relation-generator.html
+summary: summary
+---
+
+### Prerequisites
+
+Important: Before running this generator, you must create an application using
+the application generator. Then you must run the command from the root directory
+of the application. The models involved in the relation must exist before
+running this generator. The datasource involved in the relation must exist
+before running this generator. The repositories involved in the relation must
+exist before running this generator.
+
+{% include content/generator-create-app.html lang=page.lang %}
+
+### Synopsis
+
+Adds a new `Relation` between existing source and target models in a LoopBack
+application.
+
+```sh
+lb4 relation [options]
```

Comment:
Can you also add this to the [@loopback/cli README](https://github.com/strongloop/loopback-next/blob/master/packages/cli/README.md)?

Reply:
Added